api/lanes.py

```diff
@@ -562,3 +590,59 @@ def apply_filters(self, qu, work_model=Work, *args, **kwargs):
         qu = qu.join(work_edition).filter(work_edition.series==self.series)
         qu = qu.order_by(work_edition.series_position, work_edition.title)
         return qu
+
+
+class ContributorLane(QueryGeneratedLane):
+    """A lane of Works written by a particular contributor"""
+
+    ROUTE = 'contributor'
+    MAX_CACHE_AGE = 48*60*60    # 48 hours
+
+    def __init__(self, _db, contributor_name, contributor_id=None):
```

Comment:
Yes, that sounds great. I'd still like to move towards VIAF to avoid the problem of combining books by different people with the same name. But long term, like I said.